helmfile/getContext.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Let's add a file header to explain the usage of this file. 🙏

Reply:
Done!